pkg/k8shandler/certificates.go

```diff
@@ -97,6 +98,14 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCertificates() (err e
 				return err
 			}
 		}
+		for _, r := range result {
+			strs := r.([]interface{})
+			for _, s := range strs {
+				if s == "reason" {
+					clusterRequest.EventRecorder.Event(clusterRequest.Cluster, "Normal", "Updated", fmt.Sprint(strs))
```

Comment:
Is there a potential for an error here that we should at least capture and log?

Reply:
Function `Event` doesn't return any value or errors. But, I think I missed the interface conversion check at line 102